### PR TITLE
[full-ci] Make dependencies between modules more explicit and resolve some of them

### PR DESCRIPTION
**Pull request by dschmidt**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-client/src/helpers/resource/functions.ts [hunk 1]
```diff
@@ -1,4 +1,11 @@
-import { urlJoin } from 'web-pkg/src/utils'
+import { basename } from 'path'
+import { urlJoin } from '../../utils'
+import { DavPermission, DavProperty } from '../../webdav/constants'
+import { Resource } from './types'
+
+const fileExtensions = {
```
Comment (kulmann) [inline packages/web-client/src/helpers/resource/functions.ts:6]: Why did you move `fileExtensions` to `functions` instead of `types`? 🤔

Comment (dschmidt) [reply packages/web-client/src/helpers/resource/functions.ts:6]: Probably because I moved the functions going along with it here, it's not exported after all.

Have no strong preference. If you prefer it in `types` I can move it there, but it's not really a "type", is it? (I mean technically yes, but nothing you'd use in an interface or something like that)

Comment (kulmann) [reply packages/web-client/src/helpers/resource/functions.ts:6]: Hmm true, not exported. Then it's fine

**Review by kulmann — APPROVED** [review]

**Comment by dschmidt** [comment]
Yay 🤓
